homeassistant/components/system_bridge/binary_sensor.py

```diff
@@ -26,7 +26,26 @@ class SystemBridgeBinarySensorEntityDescription(BinarySensorEntityDescription):
     value: Callable = round
 
 
+def camera_in_use(data: SystemBridgeCoordinatorData) -> bool | None:
+    """Return if the camera is in use."""
+    if data.system.camera_usage is not None:
+        return len(data.system.camera_usage) > 0
```

Comment:
The code suggests the function tests if *any* camera is in use, is that the case? If so, update the docstring to:

```suggestion
    """Return if any camera is in use."""
    if data.system.camera_usage is not None:
        return len(data.system.camera_usage) > 0
```